pkg/security/apparmor/validate.go

```diff
@@ -48,20 +44,11 @@ func NewValidator() Validator {
 	if err := validateHost(); err != nil {
 		return &validator{validateHostErr: err}
 	}
-	appArmorFS, err := getAppArmorFS()
-	if err != nil {
-		return &validator{
-			validateHostErr: fmt.Errorf("error finding AppArmor FS: %v", err),
-		}
-	}
-	return &validator{
-		appArmorFS: appArmorFS,
-	}
```

Comment:
It looks like with this change `Validator.ValidateHost` interface becomes too trivial to keep. I'd suggest to remove it as well.

Reply:
Hi @bart0sh, since some tests were panicking, after discussing with SergeyKanzhelev, I made a few changes and the interface is being used.